Revise ROSCon workshop details and add note about ROSCon presentations.

destogl opened the pull request:
Updated the ROSCon workshop title to include 'Control Workshop' and specify the location as 'ROSCon 2025 Singapore'. Added a note to check ROSCon presentations for updates.

destogl committed:
Revise ROSCon workshop details and add note

Updated the ROSCon workshop title to include 'Control Workshop' and specify the location as 'ROSCon 2025 Singapore'. Added a note to check ROSCon presentations for updates.

christophfroehlich commented on doc/resources/resources.rst:
```diff
@@ -11,8 +12,8 @@ Any files submitted to the documentation should be "licensed" by stating your na
-2025-09-26 ROSCon UK Workshop: Writing Custom Robot Drivers
-,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+2025-09-26 ROSCon UK Workshop: Writing Custom Robot Drivers and Control Workshop at ROSCon 2025 Singapore
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```
not sure what this toctree below means, but this links to the wrong workshop page? 2024 instead of 2025

destogl replied:
yes I have to update it all....

destogl committed:
Added presentations files.

destogl committed:
Update resources.rst with ROSCon 2024 and 2025 details

Added presentations for ROSCon 2024 and 2025, including summaries and download links.

christophfroehlich committed:
Remove from navigation bar

christophfroehlich committed:
Merge branch 'rolling' into roscon-workshop-and-presentations
